packages/mdc-textfield/outline/index.js

```diff
@@ -52,6 +52,12 @@ class MDCTextFieldOutline extends MDCComponent {
         const path = this.root_.querySelector(strings.PATH_SELECTOR);
         path.setAttribute('d', value);
       },
+      getIdleOutlineStyleValue: (propertyName) => {
+        const idleOutlineElement = this.root_.parentNode.querySelector(strings.IDLE_OUTLINE_SELECTOR);
```

Comment:
This is the only real funkiness. Since the idle outline is a sibling of the outline root, we need to access the parent node to be able to query it.